src/Broker/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+project(Broker VERSION 1.0.0 LANGUAGES C)
+
+add_library(Broker STATIC)
+target_sources(Broker
+    PRIVATE     Broker.c
+    PUBLIC      Broker.h)
+
+target_include_directories(Broker
+  PUBLIC      ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_link_libraries(Broker
+  PRIVATE     common_build_options
+              TASTE::RuntimeMocks
+              TASTE::Packetizer)
```

Comment:
I think `Packetizer` should be marked `PUBLIC` as we expose SpacePacket defines in `Broker.h`

Reply:
Hmm, It is not a problem w kazoo, however, indeed it is exposed so potentialy other targets needs to see includes.

Reply:
done